README.md

```diff
@@ -1,2 +1,15 @@
 # vocab-handling
 Code for processing external vocabularies for Neurobagel
+
+## Preparation
+
+Before you can start extracting the terms we use from existing
+vocabularies, you first have to obtain the vocabulary files.
+The easiest way we have found so far to do this is to 
+
+1. Make an account on https://athena.ohdsi.org/search-terms/start
+2. Naviage to https://athena.ohdsi.org/vocabulary/list and select the vocabularies you want to download
```

Comment:
```suggestion
2. Navigate to https://athena.ohdsi.org/vocabulary/list and select the vocabularies you want to download
   - In the download window, provide an alias for your selected vocab collection under "name bundle"
```

since this was something that tripped me up when I tried to redownload SNOMED :)